source/product-stories.html.erb

```diff
@@ -27,6 +27,11 @@ product_stories:
     link: "/product-stories/southwark-ccg/"
     image: "/assets/images/product-stories/southwark-ccg.jpg"
 
+  - client: "char.gy"
+    description: "Providing residents with an on-street electric vehicle lamppost charging solution with char.gy"
```

Comment:
Nitpick: this should probably be `lamp post` throughout.

English Oxford Dictionary only lists the two word version https://en.oxforddictionaries.com/definition/lamp_post